### ISSUE TITLE
Listbox together with poll_external_updates crash

### ISSUE THREAD

**Issue by gonzygonz** [chriskiehl/Gooey#350]
Hello!

some info:
 - [ windows 10 ] OS
 - [ 3.6.5 ] Python Version 
 - [ 1.0.2 ] Gooey Version 

my problem:
Listbox together with poll_external_updates crash on message:
**File "C:\Python36-32\lib\site-packages\gooey\gui\components\widgets\listbox.py", line 20, in setOptions
    self.widget.SetChoices()
AttributeError: 'ListBox' object has no attribute 'SetChoices'**

if i remove the poll_external_updates=true then it works (but without the live updates of course)

i took the dynamic_updates.py from the examples and tried to change into listbox and it crashed.

basically i took the example and changed the dropdown to a listbox, and then had to adjust a few lines for it to work without poll_external_updates, and after it worked i added poll_external_updates=true and it crashed:

```
"""
Dynamic updates in Gooey

Simple example demonstrating feeding Gooey a dynamic list of file names at runtime
"""
import datetime
import json
import os
import sys

from gooey import Gooey, GooeyParser


directory_error = (
    'Unable to create a folder for your save files!\n'
    'Make sure you\re running in a directory where you have write permissions!'
)

@Gooey(
    program_name="Using Dynamic Values",
    poll_external_updates=True)
def main():
    parser = GooeyParser(description='An example of polling for updates at runtime')
    g = parser.add_argument_group()
    stuff = g.add_mutually_exclusive_group(
        required=True,
        gooey_options={
            'initial_selection': 0
        }
    )
    stuff.add_argument(
        '--save',
        metavar='Save Progress',
        action='store_true',
        help='Take a snap shot of your current progress!'
    )
    stuff.add_argument(
        '--load',
        metavar='Load Previous Save',
        help='Load a Previous save file',
        dest='filename',
        widget='Listbox',
        nargs='+',
        choices=list_savefiles(),
        gooey_options={
            'validator': {
                'test': 'user_input != "Select Option"',
                'message': 'Choose a save file from the list'
            }
        }
    )

    args = parser.parse_args()

    if args.save:
        save_file()
    else:
        for f in args.filename:
            read_file(os.path.join('saves', f))
            print('Finished reading file %s!' % f)


def read_file(path):
    with open(path, 'r') as f:
        print(f.read())


def list_savefiles():
    """ List all available files in the save directory """
    return list(sorted(os.listdir('saves'), reverse=True))


def show_error_modal(error_msg):
    """ Spawns a modal with error_msg"""
    # wx imported locally so as not to interfere with Gooey
    import wx
    app = wx.App()
    dlg = wx.MessageDialog(None, error_msg, 'Error', wx.ICON_ERROR)
    dlg.ShowModal()
    dlg.Destroy()


def save_file():
    """Save a plain ol' text file with some metadata in the title"""
    next_number = '{:04d}'.format(len(list_savefiles()) + 1)
    now = datetime.datetime.now().strftime('%b %d, %Y - %H.%M%p')
    filename = 'Save{}{}.save'.format(next_number, now)
    with open(os.path.join('saves', filename), 'w') as f:
        f.write("Hello World!")
    print('Saved {}!'.format(filename))


def mk_savedir():
    """
    Attempt to create a directory where we can store the user's save files
    """
    try:
        os.mkdir('saves')
    except IOError as e:
        if not e.winerror == 183:  # already exists
            show_error_modal(directory_error)
            sys.exit(1)


if __name__ == '__main__':
    if 'gooey-seed-ui' in sys.argv:
        print(json.dumps({'--load': list_savefiles()}))
    else:
        mk_savedir()
```

P.S this is my first time issuing an issue on github - so please be patient with me :)
 

Thanks!

**Comment by chriskiehl** [comment]
You filled out the issue template, so you did A-OK on your first Github issue ^_^ 

This'll be fixed as part of the upcoming 1.0.x release.

**Comment by gonzygonz** [comment]
Hi again.
i think this fixes this problem.
#352 
BTW - is there a schedule for the next release?

**Comment by chriskiehl** [comment]
Thanks again for fixing this! 

No official schedule on the release. I work on it when time/energy permits. However, I've added most of what I've set out to add for 1.0.3. I'm mostly just tacking on bug fixes at this point, updating docs, (which causes quite a bit of release delay due to my procrastination to avoid doc writing), updating example repos, and then, finally, releasing it out to the world. 

Official timeline: _soon_. Shooting for before the new year.
